src/api/frame/frame.ts

```diff
@@ -1,3 +1,4 @@
+declare var fin: any;
 import { Base, EmitterBase } from '../base';
```

Comment:
can you move fin related item to the openfin render api file under environment? so no need to define here.